server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 5000;
+
+// console.log that your server is up and running
+app.listen(port, () => console.log(`Listening on port ${port}`));
+
+// create a GET route
+app.get('/express_backend', (req, res) => {
+  res.send({ express: 'YOUR EXPRESS BACKEND IS CONNECTED TO REACT' });
```

Comment:
Typically you want to be explicit about the status code you're sending.

`res.status(200).send(...`

Reply:
changed to 'response.status == 200'